test/test_cli.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+import os.path as osp
+import re
+import pytest
+import sys
+
+
+def get_testdir():
+    filename = osp.normcase(osp.dirname(osp.abspath(__file__)))
+    return osp.realpath(filename)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires Python 3.7 or higher")
+def test_cli():
+    script_file = osp.join(get_testdir(), "data", "script.m")
+
+    # asserts output contains 'Hello' and '2'
+    assert re.match(
+        r"Hello[ \n\t]*2",
```

Comment:
On WIndows `\r` is needed, but probably better to use `\s`

Reply:
Yes, makes sense.